server/sql/migrations/00005_add_and_alter_columns_to_users_and_subscriptions.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE users
+  ADD user_uuid UUID NOT NULL DEFAULT gen_random_uuid();
```

Comment:
can just one-liner this. also let's be explicit about what's being added; in this case it's a new column

```suggestion
ALTER TABLE users ADD COLUMN user_uuid UUID NOT NULL DEFAULT gen_random_uuid();
```

Reply:
same comments apply to the queries below